playground/src/editor.tsx

```diff
@@ -147,6 +258,17 @@ export function Editor(props: {
     };
   }, []);
 
+  useEffect(() => {
+    props.languageService.addEventListener("diagnostics", (evt) =>
+      onCheck(evt.detail.diagnostics)
+    );
```

Comment:
In useEffect, you should return a function that is called when the component is destroyed. Here you should probably return a function that calls removeEventListener when the component is destroyed. (See line 143/254 above)

Reply:
Done